server-sent-events/README.md

```diff
@@ -0,0 +1,3 @@
+Note about runing this example through **NGINX**:
```

Comment:
Thanks for the addition! Couple of change requests:

* Typo - runing -> running
* Can you turn this line into a heading?